packages/components/src/tag/src/Tag.tsx

```diff
@@ -81,7 +81,7 @@ export function InnerTag({
         text: {
             className: "o-ui-tag-text",
             color: "inherit",
-            size: sizeValue
+            size: "md"
```

Comment:
So you don't want a tag text to accept a responsize size value?

Reply:
It has been decided that the current font size on small tags is too small. md is the minimum a tag text should be only the height changes between a small and medium tag now.